apps/zui/src/app/lakes/root.tsx

```diff
@@ -35,7 +36,7 @@ export function InitLake({children}) {
 
   switch (status) {
     case "disconnected":
-      return <ConnectionError onRetry={() => dispatch(initCurrentTab())} />
+      return <ConnectionError onRetry={() => dispatch(updateStatus(lake.id))} />
```

Comment:
`initCurrentTab` only called this `updateStatus` function. So I inlined it and removed `initCurrentTab`. 

All of this "lake" status/authentication code needs another look at some point.